web/viewer.css

```diff
@@ -1923,22 +1887,12 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
   .visibleSmallView {
     display: inherit;
   }
-  html[dir='ltr'] #outerContainer.sidebarMoving .outerCenter,
```

Comment:
Fixed in the new commit. You're right that it should be there, and it was an oversight on my part.